app/models/role.rb

```diff
@@ -1,6 +1,9 @@
 class Role < ActiveRecord::Base
   belongs_to :resource, polymorphic: true
   has_and_belongs_to_many :users, join_table: :users_roles
+
+  has_paper_trail on: [:update], only: [:name, :description], meta: {conference_id: :resource_id}
```

Comment:
Actually this only tracks change of description of role.
Tracking adding/removing users (toggle_user) is a bit complicated and I'll add that in another PR.